skimage/morphology/extrema.py

```diff
@@ -81,9 +81,9 @@ def h_maxima(image, h, footprint=None):
 
     See Also
     --------
-    skimage.morphology.extrema.h_minima
-    skimage.morphology.extrema.local_maxima
-    skimage.morphology.extrema.local_minima
+    :func:`skimage.morphology.h_minima`
+    :func:`skimage.morphology.local_maxima`
+    :func:`skimage.morphology.local_minima`
```

Comment:
```suggestion
    skimage.morphology.h_minima
    skimage.morphology.local_maxima
    skimage.morphology.local_minima
```
Same here and elsewhere in the file... The only reason this didn't work previously was because the path was wrong.